src/control/server/server_utils.go

```diff
@@ -291,7 +298,9 @@ func prepBdevStorage(srv *server, iommuEnabled bool) error {
 		// allocation as some overhead may result in one or two being unavailable.
 		prepReq.HugePageCount = srv.cfg.NrHugepages / len(numaNodes)
 		prepReq.HugePageCount += common.ExtraHugePages
-		prepReq.HugeNodes = strings.Join(numaNodes, ",")
+		prepReq.HugeNodes = strings.Trim(
+			strings.Join(strings.Split(fmt.Sprint(numaNodes), " "), ","),
+			"[]")
```

Comment:
🤨 ... You're tying this functionality to the current implementation of how the language prints an integer slice. Is it likely to change? No. But I don't think the language compatibility guarantee extends to `fmt.Sprint()` output.

It looks like the only place you're using the `numaNodes` value returned by `getEngineNUMANodes()` is here, so why not just generate the string slice in that function while you're iterating over the nodes anyhow?

Reply:
that's what it's used to do and I will change back, I was going back on forth on this myself

Reply:
done